wiki/frappe_wiki/doctype/wiki_document/wiki_document.py

```
@@ -550,6 +550,36 @@ def get_page_data(route: str) -> dict:
 	return doc.get_web_context()
 
 
+def on_wiki_document_update(doc, method):
+	"""Sync desk edits to the revision system so CRs stay aligned with the live tree."""
+	_sync_document_to_revision(doc)
+
+
+def on_wiki_document_trash(doc, method):
```

Ruff (0.15.0)

[warning] Unused function argument: `method`

(ARG001)